### PR TITLE
feat: themes page implementation

### LINKED ISSUE #55: [Task] Implement Themes Page
Task Description:
Implement the Themes page

Deliverable(s):
Implement the themes page according to design specs that is well-styled, responsive, and adheres to the correct folder structure.

Additional Context:
The Themes page mockup can be found on [mockup](https://www.figma.com/design/oK12zZACqvlACoqkZIrtxV/Gallery-Website?node-id=2545-3326&node-type=section)

### PR DESCRIPTION
**Pull request by wlee261**
#### Description
Themes page according to figma specs

Closes #55 

#### What change does this PR introduce?
Implements a themes page according to figma specs

Dark and light theme preview desktop
Screenshot 2024-09-18 112652
Screenshot 2024-09-18 112715

Dark and light theme desktop
Screenshot 2024-09-18 112724
Screenshot 2024-09-18 112733

Dark and light theme mobile
Screenshot 2024-09-18 112837
Screenshot 2024-09-18 112847

Dark and light theme preview mobile
Screenshot 2024-09-18 112857
Screenshot 2024-09-18 112909

Please select the relevant option(s).

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] CI/CD (updates related to the CI/CD process)
- [ ] Documentation update (changes to docs/code comments)
- [ ] Chore (miscellaneous tasks that do not fall into the above options)

#### What is the proposed approach?

Please give a short overview/explanation on the approach taken to resolve the issue(s).

#### Checklist:

- [ ] The commit message follows our adopted [guidelines](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Testing has been done for the change(s) added (for bug fixes/features)
- [ ] Relevant comments/docs have been added/updated (for bug fixes/features)

### CODE REVIEW

File: src/components/NavigationBar/NavigationBar.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ const NavigationBar = () => {
 
 	return (
 		<nav
-			className="fixed top-0 w-full z-50 opacity-80 bg-black
```
Comment (HusseinSerag) [inline src/components/NavigationBar/NavigationBar.tsx:79]: This change is already fixed in another PR, other than that everything is fine!

**Review by HusseinSerag — COMMENTED** [review]
Navigation bar is already fixed in another PR.

File: src/components/NavigationBar/NavigationBar.tsx (hunk 1 above)
Comment (wlee261) [reply src/components/NavigationBar/NavigationBar.tsx:79]: would you happen to know if these lint errors were left in intentionally? am I good to push up this commit (which is taking latest main) with no verify or should we look to fix these up?

Screenshot 2024-10-03 175032

Comment (tjtanjin) [reply src/components/NavigationBar/NavigationBar.tsx]: Thanks @wlee261 for the PR and @HusseinSerag for helping with the review 😊

**Review by tjtanjin — APPROVED** [review]

**Pull request merged**